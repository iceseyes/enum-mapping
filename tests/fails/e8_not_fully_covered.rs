use enum_mapping_macro::U8Mapped;

#[derive(U8Mapped)]
enum Failure {
    A(u8, u8, u8),
    B,
    C { test: String },
}

fn main() {
    let f = Failure::from(4);
}
