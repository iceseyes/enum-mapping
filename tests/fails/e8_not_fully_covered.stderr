error[E0308]: mismatched types
  --> ../tests/fails/e8_not_fully_covered.rs:11:27
   |
11 |     let f = Failure::from(4);
   |             ------------- ^ expected `Failure`, found integer
   |             |
   |             arguments to this function are incorrect
   |
note: associated function defined here
  --> $RUST/core/src/convert/mod.rs
   |
   |     fn from(value: T) -> Self;
   |        ^^^^
